backend/hitas/views/apartment.py

```diff
@@ -137,6 +151,11 @@ def create_links(instance: Apartment) -> Dict[str, Any]:
     }
 
 
+class ApartmentImprovementSerializer(serializers.Serializer):
+    market_price_index = ImprovementSerializer(many=True, source="market_price_improvements")
+    construction_price_index = ImprovementSerializer(many=True, source="construction_price_improvements")
```

Comment:
Is the `depreciation_percentage` field not needed here?

Reply:
yep, it's not needed :)